domains/s3.is-local.org.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "s3",
+    "domain": "is-local.org",
+    "subdomain": "cdns3",
```

Comment:
Subdomain doesn't matches file name.

File name: s3.is-local.org.json
Subdomain being requested: cdns3.is-local.org